commonui/src/test/java/com/hyperwallet/android/ui/common/util/TransferMethodUtilsTest.java

```diff
@@ -140,7 +141,7 @@ public void testGetStringFontIcon_returnsDefaultValue() {
         String stringResource = TransferMethodUtils.getStringFontIcon(mContext,
                 "some type");
 
-        assertThat(stringResource, is("bank icon"));
+        MatcherAssert.assertThat(stringResource, Matchers.is("bank icon"));
```

Comment:
please import the whole static class for `MatcherAssert` and `Matchers` please

Reply:
removed